specs-blocks/gutenberg-markdown-block-spec.js

```diff
@@ -70,33 +77,49 @@ if ( screenSize !== 'mobile' ) {
 				return await wpAdminSidebar.selectNewPost();
 			} );
 
-			step( 'Can insert a markdown block', async function() {
-				const gHeaderComponent = await GutenbergEditorHeaderComponent.Expect( driver );
-				await gHeaderComponent.removeNUXNotice();
-				this.markdownBlockID = await gHeaderComponent.addBlock( 'Markdown' );
-			} );
-
-			step( 'Can fill markdown block with content', async function() {
-				this.markdownBlock = await MarkdownBlockComponent.Expect( driver, this.markdownBlockID );
-				return await this.markdownBlock.setContent(
-					'### Header\nSome **list**:\n\n- item a\n- item b\n- item c\n'
-				);
-			} );
+			markdownTest();
+		} );
 
-			step( 'Can see rendered content in preview', async function() {
-				await this.markdownBlock.switchPreview();
-				const html = await this.markdownBlock.getPreviewHTML();
-				assert.equal( html, expectedHTML );
-				await this.markdownBlock.switchMarkdown();
+		describe( '[Calypso] Publish a simple post with Markdown block', function() {
+			step( 'Can login to WPCOM and start new post', async function() {
+				const loginFlow = new LoginFlow( driver, 'jetpackConnectUser' );
+				await loginFlow.login();
+				const siteSlug = dataHelper.getSiteSlug( jnFlow.url );
+				const url = `https://wpcalypso.wordpress.com/gutenberg/post/${ siteSlug }`;
+				driver.get( url );
 			} );
 
-			step( 'Can publish the post and see its content', async function() {
-				const gHeaderComponent = await GutenbergEditorHeaderComponent.Expect( driver );
-				await gHeaderComponent.publish( { visit: true } );
-				const postFrontend = await PostAreaComponent.Expect( driver );
-				const html = await postFrontend.getPostHTML();
-				assert( html.includes( expectedHTML ) );
-			} );
+			markdownTest();
 		} );
 	} );
 }
+
+function markdownTest() {
```

Comment:
Not sure if that clear enough, and is this is the best way of code re-use. But tl; dr: it inserts all these steps everywhere the function was called.

Reply:
Would renaming it make it more clear? E.g. `markdownTestSteps` or `runMarkdownTest`.